src/panels/config/zwave/zwave-node-config.ts

```diff
@@ -0,0 +1,390 @@
+import "@polymer/paper-dropdown-menu/paper-dropdown-menu";
+import "@polymer/paper-input/paper-input";
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-listbox/paper-listbox";
+
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+  PropertyValues,
+} from "lit-element";
+
+import { haStyle } from "../../../resources/styles";
+import { HomeAssistant } from "../../../types";
+
+import "../../../components/buttons/ha-call-service-button";
+import "../../../components/ha-card";
+import {
+  ZWaveConfigItem,
+  ZWaveNode,
+  ZWaveConfigServiceData,
+  fetchNodeConfig,
+} from "../../../data/zwave";
+
+@customElement("zwave-node-config")
+export class ZwaveNodeConfig extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public nodes: ZWaveNode[] = [];
+  @property() public config: ZWaveConfigItem[] = [];
+  @property() public selectedNode: number = -1;
+  @property() private _configItem?: ZWaveConfigItem;
+  @property() private _wakeupInput: number = -1;
+  @property() private _selectedConfigParameter: number = -1;
+  @property() private _selectedConfigValue: number | string = -1;
+
+  protected render(): TemplateResult | void {
+    return html`
+      <div class="content">
+        <ha-card
+          header=${this.hass!.localize(
+            "ui.panel.config.zwave.node_config.header"
+          )}
+        >
+          ${this.nodes[this.selectedNode].attributes.hasOwnProperty(
+            "wake_up_interval"
+          )
+            ? html`
+                <div class="card-actions">
+                  <paper-input
+                    float-label="${this.hass!.localize(
+                      "ui.panel.config.zwave.common.wakeup_interval"
+                    )}"
+                    type="number"
+                    .value=${this._wakeupInput !== -1
+                      ? this._wakeupInput
+                      : this.hass!.localize(
+                          "ui.panel.config.zwave.common.unknown"
+                        )}
+                    @value-changed=${this._onWakeupIntervalChanged}
+                    placeholder=${this.nodes[this.selectedNode].attributes
+                      .wake_up_interval
+                      ? this.nodes[this.selectedNode].attributes
+                          .wake_up_interval
+                      : this.hass!.localize(
+                          "ui.panel.config.zwave.common.unknown"
+                        )}
+                  >
+                    <div suffix="">
```

Comment:
Leftover from conversion

```suggestion
                    <div suffix>
```